### ISSUE TITLE
How should users deal with Xcode updates?

### ISSUE THREAD

**Issue by Ashton-W** [supermarin/Alcatraz#262]
As a user, what should you be doing after an Xcode update to get all your plugins updated and installed again.

I notice that somehow Alcatraz stays installed and available in the menubar. But looking at the Installed Plugins list I see a bunch of plugins that aren't actually working. I can click 'Remove' then 'Install' again on them to hopefully update to a version that supports the latest Xcode.

Is there an 'update all' feature I am missing? Am I using Alcatraz wrong?

**Comment by jurre** [comment]
Alcatraz automatically updates plugins when you restart xcode, however, plugins themselves do need to add support for new versions of xcode.

**Comment by Ashton-W** [comment]
I wasn't aware of that. Guess I never tried restarting Xcode again.

Should Alcatraz show a message to the user somehow to inform them of the status of pending updates?

**Comment by fabriph** [comment]
Hello everyone. I just updated to Xcode 6.3.2 and Alcatraz is not shown, also all the plugins installed are not available.
Maybe the new update breaks something in Alcatraz?

**Comment by jurre** [comment]
Could you try reinstalling alcatraz using the install script?

**Comment by xLegoz** [comment]
Just to update, Alcatraz seems to have been removed but works fine once reinstalled.

After reinstalling it with the install script, fully exiting xcode, and a prompt of "Unexpected Bundle... bundle not from Apple, Load Bundle?" it's back to normal, all plugins kept :)

**Comment by fabriph** [comment]
@jurre @xLegoz Yes, solved. Thanks.

**Comment by supermarin** [comment]
This is a dupe of some previous issues, so closing the auto-update one for now

**Comment by jostster** [comment]
this seems to be an ongoing problem. When an update is in the App Store for Xcode, alcatraz gets overwritten (or so it seems) and cannot be access from the menu. I must then reinstall alcatraz then reinstall all previous plugins.

**Comment by aledalgrande** [comment]
Confirming the problem, every time I have to restore it.

**Comment by tikipatel** [comment]
```
When an update is in the App Store for Xcode, alcatraz gets overwritten (or so it seems) and cannot be access from the menu. I must then reinstall alcatraz then reinstall all previous plugins.
```
Happens every single time.

**Comment by fontno** [comment]
Same here on previous updates and the 7.3 update.

- Open Xcode after update and clicked load bundles. 
- Package manager is available and shows plugins installed, but they are not available although they are in the plugins folder.
- Restart Xcode and select load bundles for the second time, however only some of the plugins are actually available to use.
- Go through list to remove and re-install the rest. These are immediately available and all work normally.

Additionally, I never saw any message from Alcatraz as mentioned at the beginning of this issue.

I'm happy to help out if anyone has any suggestions to resolve this issue.

**Comment by guillaume-algis** [comment]
@fontno There's two issues here: this first one will be addressed by #443, the second one is that there is no visual feedback when Alcatraz is updating plugins. So the multiple "Load Bundles" / Restart Xcode dance could have been caused by Alcatraz not having enough time to update all your plugins before the restart.

**Comment by alanhamlett** [comment]
Related to #440
